source/query-with-natural-language/prompt-natural-language-query.txt

```diff
@@ -9,6 +9,100 @@ Prompt your First Natural Language Query
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 2
+   :depth: 1
 
-Prompt your First Natural Language Query
\ No newline at end of file
+You can use MongoDB Compass to generate filter documents using the
+natural language querying feature. This feature automatically generates 
+filter documents based on prompts you provide to the MongoDB AI engine. 
+Natural language querying serves as a helpful starting point and assists 
+you in learning to write MongoDB queries.
+
+About this Task
+---------------
+
+.. note::
+
+   .. include:: /includes/fact-natural-language-query.rst
+
+- The MongoDB AI engine can return both filter documents and 
+  aggregations. If your prompt results in an aggregation, a pop-up menu
+  directs you to the :guilabel:`Aggregations` tab.
+
+- This page uses the :ref:`sample_mflix.movies <sample-mflix>` 
+  collection from the Atlas sample dataset.
+
+Before you Begin
+----------------
+
+You must :ref:`enable the natural language querying feature<enable-natural-language-querying>`.
+
+Steps
+-----
+
+.. procedure::
+   :style: connected
+
+   .. step:: Navigate to the :guilabel:`Natural Language Query Bar`
+      
+      I. Select the :guilabel:`Documents` tab.
+      #. Click the :guilabel:`Ask AI` button.
+
+      The :guilabel:`Natural Language Query Bar` displays.
+
+   .. step:: Type a question for your collection
+
+      Type a natural language prompt for your collection into the query 
+      bar. For example: ``Which movies were released in 2000?``
+
+      I. Press enter or click :guilabel:`Generate`.
+      #. A filter document populates in the :guilabel:`Filter` bar.
+
+   .. step:: Run the query
+
+      I. Before running the query, make sure to thoroughly review the 
+         syntax in the :guilabel:`Filter` bar.
+      #. Click the :guilabel:`Find` button to execute the query.
+
+      The results populate in the documents view.
```

Comment:
This also feels like a good place to encourage users to rate their query with the in-product feedback mechanisms (while also reminding them that these results won't be used to train any models)